### PR TITLE
Docs: adding veterans use case

### LINKED ISSUE #1587: Update docs site with Veteran use case
Add a new page under the _Use Cases_ section: https://docs.calitp.org/benefits/use-cases/

### PR DESCRIPTION
**Pull request by indexing**
Closes #1587

### CODE REVIEW

File: docs/use-cases/Veterans.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+**Actor:** A US veteran who uses public transit in California. For benefit eligibility, a veteran is defined as “a person who served in the active military, naval, or air service, and was discharged or released therefrom under conditions other than dishonorable.” ([source](https://www.ssa.gov/OP_Home/comp2/D-USC-38.html))
+
+**Goal:** To verify a transit rider’s veteran status and enable the rider to receive reduced fares when paying by contactless debit or credit card.
+
+**Precondition:** The California transit provider delivering fixed route service has installed and tested validator hardware necessary to collect fares using contactless payment on bus or rail lines, and the provider has a policy to offer a transit discount for US veterans.
+
+## **Basic Flow**
+
+![](https://t8631512.p.clickup-attachments.com/t8631512/5bc012c2-d4e5-4c44-8435-85dbe2f9efd8/Screenshot%202023-09-19%20at%203.58.29%20PM.png)
```
Comment (angela-tran) [inline docs/use-cases/Veterans.md:15]: Let's put the Mermaid code here

File: docs/use-cases/Veterans.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+The transit provider serving their part of California implements contactless payments on fixed bus routes throughout the service area. This rider uses [benefits.calitp.org](https://app.clickup.com/8631512/v/dc/87d6r-10994/benefits.calitp.org) to confirm their veteran status and register their debit card for reduced fares. They tap to pay when boarding buses in their area and are automatically charged the reduced fare. They no longer need to carry one card to pay for transit and another for other purchases. Best of all, they have complete access to all funds in their weekly budget. If food and medication costs are higher one week, they can allocate additional funds to those areas and ride transit less.
+
+# Untitled
```
Comment (angela-tran) [inline docs/use-cases/Veterans.md:54]: Looks like this was left on accident?

Comment (angela-tran) [inline docs/use-cases/Veterans.md:54]:
```suggestion

```

File: docs/use-cases/Veterans.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+This use case describes a feature in the [Cal-ITP Benefits app](https://benefits.calitp.org) that allows US veterans who use public transit to verify their veteran status and receive reduced fares when paying by contactless debit or credit card at participating transit providers in California.
+
+**Actor:** A US veteran who uses public transit in California. For benefit eligibility, a veteran is defined as “a person who served in the active military, naval, or air service, and was discharged or released therefrom under conditions other than dishonorable.” ([source](https://www.ssa.gov/OP_Home/comp2/D-USC-38.html))
+
+**Goal:** To verify a transit rider’s veteran status and enable the rider to receive reduced fares when paying by contactless debit or credit card.
+
+**Precondition:** The California transit provider delivering fixed route service has installed and tested validator hardware necessary to collect fares using contactless payment on bus or rail lines, and the provider has a policy to offer a transit discount for US veterans.
+
+## **Basic Flow**
```
Comment (thekaveman) [inline docs/use-cases/Veterans.md:13]: I think these bold marks `**` aren't needed on headings.

**Review by thekaveman — CHANGES_REQUESTED** [review]

File: docs/use-cases/Veterans.md (hunk 3 above)
Comment (indexing) [reply docs/use-cases/Veterans.md]: I agree. That styling came with the export from Clickup. Good catch! I've removed the bolding in a new commit.

**Review by thekaveman — DISMISSED** [review]
Nice work @indexing!

**Review by angela-tran — APPROVED** [review]

**Comment by thekaveman** [comment]
@machikoyasuda I think you can merge this once you've had a chance to review. 

I'm guessing @indexing does not see the `Merge pull request` button because of permissions?

File: docs/use-cases/Veterans.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+3. The transit rider selects the option to receive a reduced fare for veterans.
+4. The transit rider authenticates with their existing [Login.gov](https://Login.gov) account or creates a [Login.gov](https://Login.gov) account if they don’t have one.
+5. The Cal-ITP Benefits app interfaces with the [California Department of Technology](https://cdt.ca.gov/) Identity Gateway (IdG) to verify benefit eligibility. The IdG requests the required personal information to verify veteran status from [Login.gov](https://Login.gov).
+6. The IdG utilizes the [Veteran Confirmation API](https://developer.va.gov/explore/api/veteran-confirmation) provided by the US Department of Veterans Affairs to determine the rider’s veteran status.
+7. The IdG passes the response from [VA.gov](https://VA.gov) as veteran status = TRUE to the Cal-ITP Benefits app to indicate the person is eligible for a benefit.
+8. The transit rider provides the debit or credit card details they use to pay for transit to Littlepay, the payment processor that facilitates transit fare collection.
+9. The app registers the veteran benefit with the transit rider’s debit or credit card.
+
+## Alternative Flows
```
Comment (machikoyasuda) [inline docs/use-cases/Veterans.md:60]: Lowecase the f. We use sentence-case.

Comment (machikoyasuda) [inline docs/use-cases/Veterans.md:60]:
```suggestion
## Alternative flows
```

File: docs/use-cases/Veterans.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+## Benefits
+
+*   The transit rider no longer needs cash to pay for transit rides.
+*   The transit rider doesn’t have to lock up funds on a closed-loop card offered by the transit provider.
+*   The transit rider pays for transit rides with their debit or credit card, just as they pay for any other good or service that accepts contactless payment.
+*   The transit rider can enroll in a transit benefit from home when convenient; they do not have to visit a transit provider in person.
+*   Secure state and federal solutions manage the transit rider’s personal identifiable information (PII): [Login.gov](https://Login.gov) and the California Department of Technology Identity Gateway (IdG). Transit riders do not have to share personal information with local transit agencies.
+*   Benefits enrollment takes minutes rather than days or weeks.
+
+## Example Scenario
```
Comment (machikoyasuda) [inline docs/use-cases/Veterans.md:81]:
```suggestion
## Example scenario
```

**Review by machikoyasuda — CHANGES_REQUESTED** [review]
2 small suggestions @indexing  (which you can just click `Commit suggestion` on) and then we're good!

**Comment by angela-tran** [comment]
> @machikoyasuda I think you can merge this once you've had a chance to review.
> 
> I'm guessing @indexing does not see the `Merge pull request` button because of permissions?
> image

@indexing Just curious, whenever the pull request is approved, do you see a "Merge pull request" button?

**Review by indexing — COMMENTED** [review]
Accepted suggestions

File: docs/use-cases/Veterans.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+This use case describes a feature in the [Cal-ITP Benefits app](https://benefits.calitp.org) that allows US veterans who use public transit to verify their veteran status and receive reduced fares when paying by contactless debit or credit card at participating transit providers in California.
+
+**Actor:** A US veteran who uses public transit in California. For benefit eligibility, a veteran is defined as “a person who served in the active military, naval, or air service, and was discharged or released therefrom under conditions other than dishonorable.” ([source](https://www.ssa.gov/OP_Home/comp2/D-USC-38.html))
+
+**Goal:** To verify a transit rider’s veteran status and enable the rider to receive reduced fares when paying by contactless debit or credit card.
+
+**Precondition:** The California transit provider delivering fixed route service has installed and tested validator hardware necessary to collect fares using contactless payment on bus or rail lines, and the provider has a policy to offer a transit discount for US veterans.
+
+## Basic Flow
```
Comment (thekaveman) [inline docs/use-cases/Veterans.md:13]:
```suggestion
## Basic flow
```

One more

**Review by thekaveman — CHANGES_REQUESTED** [review]

**Review by thekaveman — APPROVED** [review]

**Review by angela-tran — APPROVED** [review]

**Comment by thekaveman** [comment]
@machikoyasuda go ahead and merge when you approve -- I confirmed with @indexing he does not currently have the permissions.

**Review by machikoyasuda — APPROVED** [review]

**Pull request merged**